src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -8,7 +8,13 @@ import BrowserBehavior from './BrowserBehavior';
 import ExtendedBrowserBehavior from './ExtendedBrowserBehavior';
 
 export default class DefaultBrowserBehavior implements BrowserBehavior, ExtendedBrowserBehavior {
-  private readonly browser = detect();
+  private readonly FALLBACK_BROWSER = {
+    type: 'unknown',
```

Comment:
nit: Should `type` be `browser` ? From BrowserInfo that seems like the default value for browser info.
```
readonly name: Browser;
  readonly version: string;
  readonly os: OperatingSystem | null;
  readonly type = "browser";
```

Reply:
The return type of `detect()` is `BrowserInfo | SearchBotDeviceInfo | BotInfo | NodeInfo | ReactNativeInfo | null`. 
* `BrowserInfo` has `type` of `"browser"`.
* `BotInfo` has `type` of `"bot"`.
* `NodeInfo` has `type` of `"node"`.
* `ReactNativeInfo` has `type` of `"react-native"`.

Setting `type` to `"browser"` works, but I think `"unknown"` should be more appropriate. How do you think?

Reply:
Yeah I dont really have a strong opinion on it. It is just that you already call the variable `Fallback_browser` which to indicate the type to be `browser`. Also the class name is also browserbehavior.

Reply:
Yeah, let me change it to "browser". We don't use this `type` anywhere so it doesn't matter that much. And "browser" seems to be more consistent.

Reply:
Updated